lms/services/lti_grading/_v11.py

```diff
@@ -27,11 +30,13 @@ def read_result(self, grading_id) -> GradingResult:
         result = GradingResult(score=None, comment=None)
         try:
             response = self._send_request(
+                self.application_instance,
```

Comment:
send_request takes an explicit application instance and service URL.

For existing methods it will use the ones passed into the constructor.